dev-support/test-patch.sh

```diff
@@ -3345,8 +2852,13 @@ function postinstall
   local routine
   local plugin
 
+  if declare -f ${BUILDTOOL}_postapply_install >/dev/null; then
+    "${BUILDTOOL}_postapply_install"
+    (( RESULT = RESULT + $? ))
+  fi
```

Comment:
shellcheck: 18: warning: This loop will only run once, with routine='check_patch_javadoc'. [SC2043]